cmd/root.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/Xuanwo/go-locale"
+	"github.com/spf13/cobra"
+
+	"github.com/thegreenwebfoundation/grid-intensity-go/data"
+)
+
+const (
+	countryCode = "country-code"
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "grid-intensity",
+	Short: "Get carbon intensity data for electricity grids",
+	Long: `A tool for getting the carbon intensity data for electricity grids.
+
+This can be used to make your sofware carbon aware so it runs at times when the
+grid is greener or at locations where carbon intensity is lower.
+
+	grid-intensity --country-code ARG
+	grid-intensity -c BOL`,
```

Comment:
The text here is used for the help text. I tried to base it on the readme but we can easily change it.

Its recommended to have some examples at the end.